SU2_CFD/src/variable_adjoint_mean_inc.cpp

```diff
@@ -0,0 +1,261 @@
+  
+  /*--- Array initialization ---*/
+	Psi = NULL;
+	ForceProj_Vector = NULL;
+	ObjFuncSource = NULL;
+	IntBoundary_Jump = NULL;
+	TS_Source = NULL;
+  
+}
+
+CAdjIncEulerVariable::CAdjIncEulerVariable(su2double val_psirho, su2double *val_phi, su2double val_psie, unsigned short val_nDim,
+																		 unsigned short val_nvar, CConfig *config) : CVariable(val_nDim, val_nvar, config) {
+	unsigned short iVar, iDim, iMesh, nMGSmooth = 0;
+  
+  bool dual_time = ((config->GetUnsteady_Simulation() == DT_STEPPING_1ST) ||
+                    (config->GetUnsteady_Simulation() == DT_STEPPING_2ND));
+  
+  /*--- Array initialization ---*/
+	Psi = NULL;
+	ForceProj_Vector = NULL;
+	ObjFuncSource = NULL;
+	IntBoundary_Jump = NULL;
+	TS_Source = NULL;
+  
+	/*--- Allocate residual structures ---*/
+  Res_TruncError = new su2double [nVar];
+  
+  for (iVar = 0; iVar < nVar; iVar++) {
+		Res_TruncError[iVar] = 0.0;
+	}
+  
+  /*--- Only for residual smoothing (multigrid) ---*/
+	for (iMesh = 0; iMesh <= config->GetnMGLevels(); iMesh++)
+		nMGSmooth += config->GetMG_CorrecSmooth(iMesh);
+  
+  if (nMGSmooth > 0) {
+    Residual_Sum = new su2double [nVar];
+    Residual_Old = new su2double [nVar];
+  }
+	
+	/*--- Allocate undivided laplacian (centered) and limiter (upwind)---*/
+	if (config->GetKind_ConvNumScheme_AdjFlow() == SPACE_CENTERED)
+		Undivided_Laplacian = new su2double [nVar];
+	if (config->GetKind_ConvNumScheme_AdjFlow() == SPACE_UPWIND) {
+		Limiter = new su2double [nVar];
+		Solution_Max = new su2double [nVar];
+		Solution_Min = new su2double [nVar];
+		for (iVar = 0; iVar < nVar; iVar++) {
+			Limiter[iVar] = 0.0;
+			Solution_Max[iVar] = 0.0;
+			Solution_Min[iVar] = 0.0;
+		}
+	}
+  
+  /*--- Allocate and initialize solution ---*/
+  Solution[0] = 0.0; 	Solution_Old[0] = 0.0;
+  for (iDim = 0; iDim < nDim; iDim++) {
+    Solution[iDim+1] = 0.0;
+    Solution_Old[iDim+1] = 0.0;
+  }
+
+  
+  /*--- Allocate and initialize solution for dual time strategy ---*/
+	if (dual_time) {
+    Solution_time_n[0] = 0.0;
+    Solution_time_n1[0] = 0.0;
+    for (iDim = 0; iDim < nDim; iDim++) {
+      Solution_time_n[iDim+1] = 0.0;
+      Solution_time_n1[iDim+1] = 0.0;
+    }
+	}
+  
+  /*--- Allocate auxiliar vector for sensitivity computation ---*/
+	Grad_AuxVar = new su2double [nDim];
+	
+	/*--- Allocate and initialize projection vector for wall boundary condition ---*/
+	ForceProj_Vector = new su2double [nDim];
+	for (iDim = 0; iDim < nDim; iDim++)
+		ForceProj_Vector[iDim] = 0.0;
+  
+  /*--- Allocate and initialize interior boundary jump vector for near field boundary condition ---*/
+	IntBoundary_Jump = new su2double [nVar];
+	for (iVar = 0; iVar < nVar; iVar++)
+		IntBoundary_Jump[iVar] = 0.0;
+  
+  /*--- Allocate space for the time spectral source terms ---*/
+  if (config->GetUnsteady_Simulation() == HARMONIC_BALANCE) {
+		TS_Source = new su2double[nVar];
+		for (iVar = 0; iVar < nVar; iVar++)
+			TS_Source[iVar] = 0.0;
+	}
```

Comment:
There are not anymore TS source terms but now just HB sources. Can you please check this?